src/components/MovieDetails/MovieDetails.test.tsx

```diff
@@ -0,0 +1,36 @@
+import { render, screen } from '@testing-library/react';
+import { MovieContext } from '../../shared/contexts/MovieContext';
+import { formatRuntime } from '../../utils/utils';
+import MovieDetails from './MovieDetails';
+import { mockMovie } from '../../utils/mocks/movie.mock';
+
+jest.mock('../../utils/utils', () => ({
+  formatRuntime: jest.fn().mockReturnValueOnce('2h 22min'),
+}));
+
+describe('MovieDetails', () => {
+  test('renders movie details when movie is selected', () => {
+    render(
+      <MovieContext.Provider value={{ selectedMovie: mockMovie, setSelectedMovie: () => {} }}>
```

Comment:
**suggestion:** I'd personally suggest to create helper function for providers(e. g. `createWrapper()`) as in the future we can have multiple contexts

Reply:
done